### PR TITLE
issue ticket template add

### LINKED ISSUE #625: Setup the issue description template for user
## Description: 

- Currently any new user who comes up to this repository and reports a new issue ticket, we don't have a template. For a better user and contributor experience, shall we add a template for ease process?

## Potential Solution: 

- Let's setup a GitHub actions for the same? Or else we can create a md file and from settings we can manually mark that file as template. Let's discuss more on it.

### PR DESCRIPTION
**Pull request by samyakshah3008**
### Issue

Closes #625 

### What is the change?

Setup the issue ticket template for a contributor for better contributing experience. 

### Is Development Tested?

- [X] Yes
- [ ] No

### CODE REVIEW

**Review by satyam73 — DISMISSED** [review]
LGTM

**Review by rohan09-raj — APPROVED** [review]
Thanks for adding this, good work 💯 
This will definitely help all folks to write description in proper manner

File: .github/Issue_Ticket_Template.md
Comment (rohan09-raj) [inline .github/Issue_Ticket_Template.md:1]: @samyakshah3008 Can you please resolve the lint checks by using `npm run fix` ?

Comment (samyakshah3008) [reply .github/Issue_Ticket_Template.md:1]: Done

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**